ts/src/pro/bitget.ts

```diff
@@ -682,7 +682,8 @@ export default class bitget extends bitgetRest {
             stored = new ArrayCache (limit);
             this.trades[symbol] = stored;
         }
-        const data = this.safeValue (message, 'data', []);
+        let data = this.safeList (message, 'data', []);
+        data = data.reverse (); // fix chronological order
```

Comment:
@ttodua, instead of doing this, why not iterate from the end? seems more efficient

Reply:
travis throws unrelated issue